yb-voyager/src/callhome/diagnostics.go

```diff
@@ -0,0 +1,162 @@
+package callhome
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/google/uuid"
+	log "github.com/sirupsen/logrus"
+)
+
+//call-home json formats
+var (
+	jsonFilePath    string
+	Payload         payload
+	jsonBuf         []byte
```

Comment:
Why do you need the `jsonFilePath` and `jsonBuf` at the package level? Can we make them local to the function?

Reply:
I am reusing the values in different functions